imap_processing/hi/l1a/science_direct_event.py

```diff
@@ -107,14 +107,18 @@ def parse_direct_event(event_data: str) -> dict:
     metaevent = 0
     if event_type == metaevent:
         # parse metaevent
-        event_type = event_data[:2]
+        event_type = event_data[:2]  # type: ignore[assignment]
+        # TODO Change, Incompatible types in assignment
+        # (expression has type "str", variable has type "int")
```

Comment:
I suggest changing the initial declaration of `event_type` to `event_type = int(event_data[:2], 2)`. Then this line reassigning `event_type` can be removed.

```suggestion
```